AutoNode/blockchain.py

```diff
@@ -0,0 +1,103 @@
+import json
+
+import requests
+
+from .common import (
+    node_config
+)
+
+_headers = {
+    'Content-Type': 'application/json'
+}
+
+
+def get_current_epoch(endpoint=node_config['endpoint']):
+    return int(get_latest_header(endpoint)["epoch"])
+
+
+def get_latest_header(endpoint=node_config['endpoint']):
+    payload = json.dumps({"id": "1", "jsonrpc": "2.0",
+                          "method": "hmy_latestHeader",
+                          "params": []})
+    response = requests.request('POST', endpoint, headers=_headers, data=payload, allow_redirects=False, timeout=3)
```

Comment:
why redirects=false, what if the AWS endpoint does lots of redirections, why not follow it

Reply:
Changed to allow redirects as part of blockchain.py refactor